src/test/ui/const-eval/infinite_loop_hard.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(const_fn, const_let)]
+
+const fn churn() -> *const u8 {
+    loop
+    let x = 0u8;
+    &x
+}
+
+fn main() {
```

Comment:
Uh. We don't detect this loop. So our test suite will now not terminate? XD

Please remove this test

Reply:
How did that get there? An errant `stash apply` I think.